File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        //using count sort 
        int c0 =0 ;
        int c1=0 ;
        int c2 =0 ;

        for(auto i : nums){
            if(i == 0) c0++ ;
            if(i ==1) c1++ ;
            if(i ==2) c2++ ;
        }
        nums.clear() ;
        while(c0--){
            nums.push_back(0) ;
        }
        while(c1--){
            nums.push_back(1) ;
        }
        while(c2--){
            nums.push_back(2) ;
        }
    }
};

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zero,one,two = 0,0,0

        for i in nums:
            if(i==0):
                zero += 1
            elif i==1:
                one+= 1
            else:
                two += 1
        
        i =0
        while(zero):
            nums[i] = 0
            i+=1
            zero -= 1
        while(one):
            nums[i]= 1
            i+= 1
            one-=1
        while(two):
            nums[i] = 2
            i+=1
            two-= 1
            

        